npm-shrinkwrap.json

```diff
@@ -3792,24 +3792,24 @@
       }
     },
     "balena-preload": {
-      "version": "10.4.8",
-      "resolved": "https://registry.npmjs.org/balena-preload/-/balena-preload-10.4.8.tgz",
-      "integrity": "sha512-HENu1BfBW+f9VthtpentxU24inhGI1pJX1Q42Qv+iHyXoBxbd2GzCW88vvWjRbll8j+jM25PzkSABqxFRWYXMA==",
+      "version": "10.4.19",
```

Comment:
Here it resolved to `10.4.19`, while `package.json` requested `^10.4.10`. This may or may not have been the intention, and it may or may not matter. But if `10.4.19` was good, better yet would be `10.4.20`, which fixes a potential bug.

Reply:
I meant to pin to v10.4.10 but I see how I actually selected greater than.

I wasn't going to grab latest while you were on vacation, but I'm okay with it if you are!